src/ssmDocument/commands/publishDocument.ts

```diff
@@ -23,30 +26,38 @@ import * as telemetry from '../../shared/telemetry/telemetry'
 
 const DEFAULT_REGION: string = 'us-east-1'
 
-export async function publishSSMDocument(awsContext: AwsContext, outputChannel: vscode.OutputChannel): Promise<void> {
+export async function publishSSMDocument(
+    awsContext: AwsContext,
+    regionProvider: RegionProvider,
+    outputChannel: vscode.OutputChannel
+): Promise<void> {
     const logger: Logger = getLogger()
 
     const textDocument = vscode.window.activeTextEditor?.document
     if (!textDocument) {
-        let error = new Error('Could not get active text editor for local SSM Document definition')
-        logger.error(error.message)
-        throw error
+        let errorMsg = 'Could not get active text editor for local SSM Document definition'
+        logger.error(errorMsg)
+        vscode.window.showErrorMessage(
+            localize('AWS.message.error.ssmDocument.publishDocument.could_not_open', errorMsg)
+        )
+        return
     }
 
     if (textDocument.languageId !== ssmJson && textDocument.languageId !== ssmYaml) {
         let supportedFormats = [ssmJson, ssmYaml]
-        let error = new Error(
-            'Current editor language does not match the supported formats: ' + supportedFormats.join(', ')
+        let errorMsg = 'Current editor language does not match the supported formats: ' + supportedFormats.join(', ')
+        logger.error(errorMsg)
+        vscode.window.showErrorMessage(
+            localize('AWS.message.error.ssmDocument.publishDocument.invalid_format', errorMsg)
```

Comment:
This doesn't appear to use the localize call properly.

I believe it is something along the lines of 
localize(some_string_id, 'Current editor language does not match the supported formats: {0}', supportedFormats.join(', ') )